packages/babel-plugin/src/utils/css-builders.tsx

```diff
@@ -13,11 +13,24 @@ export interface CSSOutput {
   }[];
 }
 
+/**
+ * Will look in an expression and return the actual value.
+ * If the expression is an identifier node (a variable) and a constant,
+ * it will return the variable reference.
+ *
+ * E.g: If there was a identifier called `color` that is set somewhere as `const color = 'blue'`,
+ * passing the `color` identifier to this function would return `'blue'`.
```

Comment:
👏 this answers my question from the first time I reviewed this! much more approachable :)